xarray/util/deprecation_helpers.py

```diff
@@ -0,0 +1,86 @@
+import inspect
```

Comment:
I would suggest keeping the original copyright notice at the top of the this file.